src/Root.tsx

```diff
@@ -0,0 +1,28 @@
+import { App } from './App';
+import { PeoplePage } from './components/PeoplePage';
+import { HomePage } from './components/HomePage';
+import {
+  Routes,
+  Route,
+  HashRouter as Router,
+  Navigate,
+} from 'react-router-dom';
+import { ErrorPage } from './components/ErrorPage';
+
+export const Root = () => {
+  return (
+    <Router>
```

Comment:
```suggestion
export const Root = () => (
    <Router>
```

you can write it without return